vms/platformvm/txs/txheap/by_age.go

```diff
@@ -1,20 +0,0 @@
-// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
-// See the file LICENSE for licensing terms.
-
-package txheap
```

Comment:
IIRC we could drop the txheap package completelly, once the mempool sorts transactions in a FIFO order. Please cleanup all you can here.

Reply:
Found out `BuildGenesis` P-Chain API still uses it :)